src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionController.kt

```diff
@@ -19,16 +19,15 @@ import org.springframework.security.core.Authentication
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
-import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import java.util.UUID
 
-internal const val IDENTIFICATION_SESSIONS_BASE_PATH = "/api/v1/identification/sessions"
+internal const val IDENTIFICATIONS_BASE_PATH = "/api/v1/identifications"
+internal const val IDENTIFICATIONS_OLD_BASE_PATH = "/api/v1/identification/sessions"
```

Comment:
I don't like the term `OLD`. How about using `DEPRECATED` instead?

Reply:
Good point and I agree. However, as this will only be alive for a short period of time, I would leave it like this for now.